### ISSUE TITLE
86 high severity vulnerabilities after init a fastify empty project

### ISSUE THREAD

**Issue by rodmoreno** [fastify/fastify-cli#368]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure it has not already been reported

### Fastify version

3.15.1

### Node.js version

14.17.0

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

11.3.1

### Description

npm found 86 high severity vulnerabilities after `npm install` for a new fastify project created with `npm init fastify`. All those vulnerabilities are from `tap` dependency.

`npm audit fix` doesn't solve those vulnerabilities.

### Steps to Reproduce

1. On a new directory run: `npm init fastify`
2. After fastify-cli creates the project run: `npm install`

### Expected Behavior

I expect a new fastify project without vulnerabilities

**Comment by zekth** [comment]
We need to move the template to tap 15. Would you mind sending a PR for this?

**Comment by rodmoreno** [comment]
Yeah! I can do it!

Actually I already forked this repo but I notice into `generate.js` uses fastify-cli package.json for resulting new project. The problem comes when I looked into fastify-cli package.json and had `"node": ">= 6.15.0"` on the other hand tap 15 drops support for node <10.

So I'll need also update fastify-cli package.json to also drop support for node <10 and I don't know if this would have a negative impact.
